pkg/tunnelserver/tunnel.go

```diff
@@ -154,7 +154,7 @@ func (t *Authorizer) Authorize(req *http.Request) (*Client, bool, error) {
 
 func (t *Authorizer) getMachine(cluster *v3.Cluster, inNode *client.Node) (*v3.Node, error) {
 	machineName := machineName(inNode)
-	logrus.Debugf("getMachine: looking up machine [%s] in cluster [%s]", machineName, cluster.Name)
+	//logrus.Debugf("getMachine: looking up machine [%s] in cluster [%s]", machineName, cluster.Name)
```

Comment:
you probably want to uncomment this?

Reply:
Yeah, I will skip this file. For some reason, it was printing insane number of debug messages in my setup.